packages/open-lens/src/renderer/application-information.injectable.ts

```diff
@@ -0,0 +1,17 @@
+
+import { getInjectable } from "@ogre-tools/injectable";
+import { applicationInformationToken, ApplicationInformation } from "@k8slens/core/renderer";
+import packageJson from "../../package.json";
+
+const applicationInformationInjectable = getInjectable({
+  id: "application-information",
+  injectionToken: applicationInformationToken,
+  instantiate: () => {
+    const { version, config, productName, build, copyright, description, name } = packageJson;
```

Comment:
Maybe we should make getting `package.json` data as injectable? _(instead of direct import)_